### DISCUSSION TITLE
Intel Corporation Wireless 8260 channel -1 on managed/normal mode

### DISCUSSION THREAD

**Discussion by MyStuffYT** [aircrack-ng/aircrack-ng#2597] [Q&A]
So, when i use `airmon-ng start wlp4s0` it will work with `airodump-ng`, `aireplay-ng` and `airbase-ng`.
But, when i don't use Monitor Mode....
```
ioctl(SIOCSIWMODE) failed: Device or resource busy
23:51:52  Created tap interface at0
23:51:52  Trying to set MTU on at0 to 1500
23:51:52  Trying to set MTU on wlp4s0 to 1800
23:51:52  Access Point with BSSID MAC:ADDRESS started.
Error: Got channel -1, expected a value > 0.
```
No, i'm not using injection attacks.
I'll change to a normal USB adapter soon but for now i'm using my laptop's
(Yes my "old laptop" might be 2017 but it still has the job done)

**Comment by Mister-X-** [comment]
Issues are to be expected if you don't set monitor mode and don't kill network managers.
